src/CMake/PluginVsInstall.cmake.in

```diff
@@ -196,7 +196,8 @@ set(VISIT_DBIO_ONLY              @VISIT_DBIO_ONLY@)
 set(VISIT_PARALLEL               @VISIT_PARALLEL@)
 set(VISIT_JAVA                   @VISIT_JAVA@)
 set(VISIT_SLIVR                  @VISIT_SLIVR@)
-set(VISIT_OSPRAY                 @VISIT_OSPRAY@)
+set(HAVE_LIBOSPRAY               @HAVE_LIBOSPRAY@)
+set(HAVE_OSPRAY                  @HAVE_OSPRAY@)
```

Comment:
You are correct, the logic for this could be handled differently, and probably with one CMake var.
Current CMake logic doesn't use target_compile_definitions for HAVE_OSPRAY like it should, but instead uses the global 'add_definitions' in the FindOSPRay module.  The need to support building PluginsVsInstall in its current form is the reason for adding both vars here.

I can enter a new ticket to fix this, or make it a bullet point in my cmake modernization efforts.
Which would you prefer?

Reply:
Ok, sounds fine.